java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -2143,18 +2165,36 @@ public CompletableFuture<Long> pfadd(@NonNull String key, @NonNull String[] elem
         return commandManager.submitNewCommand(PfAdd, arguments, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Long> pfadd(@NonNull GlideString key, @NonNull GlideString[] elements) {
+        GlideString[] arguments = ArrayUtils.addFirst(elements, key);
+        return commandManager.submitNewCommand(PfAdd, arguments, this::handleLongResponse);
+    }
+
     @Override
     public CompletableFuture<Long> pfcount(@NonNull String[] keys) {
         return commandManager.submitNewCommand(PfCount, keys, this::handleLongResponse);
     }
 
+    @Override
+    public CompletableFuture<Long> pfcount(@NonNull GlideString[] keys) {
+        return commandManager.submitNewCommand(PfCount, keys, this::handleLongResponse);
+    }
+
     @Override
     public CompletableFuture<String> pfmerge(
             @NonNull String destination, @NonNull String[] sourceKeys) {
         String[] arguments = ArrayUtils.addFirst(sourceKeys, destination);
         return commandManager.submitNewCommand(PfMerge, arguments, this::handleStringResponse);
     }
 
+    //    @Override // Doesn't work with override for some reason!
```

Comment:
What is the error?

Reply:
Seems to work not after having merged main, removed comment